docs/standard/containerized-lifecycle-architecture/road-to-modern-applications-based-on-containers.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
we usually keep just meaningful words in the file name.

Reply:
I'd change it if possible.

Reply:
I changed in my local repo, not sure how to do it in the middle of this PR with all it's comments, is it Ok If I do it in another PR?

Reply:
Sure, let's just make sure to merge the new one quickly after this one is merged, so the page is not published live.